values.yaml

```diff
@@ -0,0 +1,30 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    repository: onetimesecret/onetimesecret
+    tag: latest
+  service:
+    type: LoadBalancer
+    port: 3000
+  env:
+    REDIS_URL: "redis://onetimesecret-redis-master.default.svc.cluster.local:6379/0"
+    COLONEL: "admin@example.com"
+    HOST: "onetimesecret-onetimesecret.default.svc.cluster.local:3000"
+    SSL: "false"  # Ensure this is a string
+
+ingress:
+  enabled: true
+  annotations: {}
```

Comment:
Can you also add support for ingressClassName?

Reply:
done..added in latest commit